app/views/courses/_description_content.html.erb

```diff
@@ -0,0 +1,127 @@
+          <%= @course.attributes[:how_school_placements_work] || 'Empty' %>
+        </dd>
+      </div>
+    </dl>
+
+    <h3 class="govuk-heading-m">
+      <% if @course.attributes[:funding] == 'fee' %>
+        <%= link_to 'Course length and fees', manage_ui_course_page_url(provider_code: @provider[:provider_code], course_code: @course.attributes[:course_code]) + '/fees-and-length', class: 'govuk-link' %>
+      <% else %>
+        <%= link_to 'Course length and salary', manage_ui_course_page_url(provider_code: @provider[:provider_code], course_code: @course.attributes[:course_code]) + '/salary', class: 'govuk-link' %>
+      <% end %>
+    </h3>
+    <dl class="govuk-summary-list govuk-summary-list--short">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key ">
+          Course length
+        </dt>
+        <dd class="govuk-summary-list__value govuk-summary-list__value--truncate <%= 'course-parts__fields__value--empty' if @course.attributes[:course_length].nil? %>" data-qa="course__length">
+          <%= course_length(@course) || 'Empty' %>
+        </dd>
+      </div>
+      <% if @course.attributes[:funding] == 'fee' %>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key ">
+            Fee for UK and EU students
+          </dt>
+          <dd class="govuk-summary-list__value <%= 'course-parts__fields__value--empty' if @course.attributes[:fee_uk_eu].nil? %>" data-qa="course__uk_fees">
+            <%= number_to_currency(@course.attributes[:fee_uk_eu]) || 'Empty' %>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key ">
+            Fee for international students (optional)
+          </dt>
+          <dd class="govuk-summary-list__value <%= 'course-parts__fields__value--empty' if @course.attributes[:fee_international].nil? %>" data-qa="course__international_fees">
+            <%= number_to_currency(@course.attributes[:fee_international]) || 'Empty' %>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key ">
+            Fee details (optional)
+          </dt>
+          <dd class="govuk-summary-list__value govuk-summary-list__value--truncate <%= 'course-parts__fields__value--empty' if @course.attributes[:fee_details].nil? %>" data-qa="course__fee_details">
+            <%= @course.attributes[:fee_details] || 'Empty' %>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key ">
+            Financial support you offer (optional)
+          </dt>
+          <dd class="govuk-summary-list__value govuk-summary-list__value--truncate <%= 'course-parts__fields__value--empty' if @course.attributes[:financial_support].nil? %>" data-qa="course__financial_support">
+            <%= @course.attributes[:financial_support] || 'Empty' %>
```

Comment:
All of these make me think that we really need to adopt a presenter pattern.

Reply:
Absolutely. I want to take another stab at getting draper in here, unless we can think of a better option.